specs/0114-pretranslation-monitoring.md

```diff
@@ -0,0 +1,46 @@
+- Feature Name: Pretranslation Monitoring
+- Created: 2023-07-05
+- Associated Issue: #2894
+
+# Summary
+
+Upgrade Insights dashboards with the ability to continuously monitor quality of pretranslations across locales and projects.
+
+# Motivation
+
+As part of the pretranslation feature testing with a limited number of locales, we used [scripts](https://github.com/mozilla-l10n/pontoon-scripts/tree/main/stats/pretranslation) to generate [evaluation sheets](https://docs.google.com/spreadsheets/d/1CH7s-ykyAq9G6vf4qizvzGUsD9pamWAM04BOOIlZzac/edit#gid=610852174) that were used to track performance of the pretranslation feature across participating locales and projects.
+
+Since the feature is now ready for larger adoption, we'd like to automate the process of collecting the data about pretranslations and show it on the dashboards. That will allow PMs to see which locales struggle with the review performance, which projects are not a good fit for pretranslation, and to make decisions accordingly.
+
+# Feature explanation
+
+A new "Time to review pretranslations" chart is made available under the "Time to review suggestions" chart in the Team Insights page. It's a line chart displaying two separate data points over the last 12 months:
+* *Current month*: Average age of pretranslations reviewed during the specific month.
+* *12-month average*: Average age of pretranslations reviewed in the 12 months before the specific month.
+
+**Text for infobox**: *How much time it takes on average to review a pretranslation.* The above bullet points should be added to the infobar to describe the plotted data points.
+
+A new "Pretranslation quality" chart is made available at the bottom of the (Team, Project and Project-Locale) Insights pages, presenting data for the period of the last 12 months. It is a combination of three charts:
+
+1. A column chart showing pretranslation review actions over time in a stack of two columns - approvals and rejections of pretranslations. Approvals and rejections performed by sync are excluded. New pretranslations are plotted in a separate column, which is hidden by default.
```

Comment:
Why the hidden column?

Reply:
Because it doesn't have impact on the quality metrics (approval ratio and chrF++ score).

It's also consistent with New suggestions and New source strings in the Review/Translation activity panels.